proto/sge/bet/bet.proto

```diff
@@ -74,6 +74,9 @@ message Bet {
   // bet_fulfillment is the fulfillment data.
   repeated BetFulfillment bet_fulfillment = 14;
 
+  // meta is metadata for bet
+  MetaData meta = 15;
```

Comment:
Do we need MetaData?

We already have sgenetwork.sge.bet.OddsType selected_odds_type = 1; in ticket.proto

Also, we have selected odds. 

Seems like both of these fields are not required

Reply:
Everything that is inside metadata, is not being processed by blockchain and is for the services, So it is a metadata in point of view of blockchain.